src/app/route/index.js

```diff
@@ -261,9 +262,12 @@ class AppRouter extends PureComponent {
 
     render() {
         return (
-            <Router history={ history }>
-                { this.renderRouterContent() }
-            </Router>
+            <>
+                <Meta />
```

Comment:
What is this for? Maybe we can leave this as is, but use global state to change meta, instead of rendering them on every page?